compose/ui/ui-backhandler/src/androidMain/kotlin/androidx/compose/ui/backhandler/BackHandler.android.kt

```diff
@@ -17,17 +17,16 @@
 package androidx.compose.ui.backhandler
 
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.ExperimentalComposeUiApi
 import kotlinx.coroutines.flow.Flow
 
-@ExperimentalComposeUiApi
```

Comment:
We must keep it as `ExperimentalComposeUiApi` even after deprecation. It should be an addition, not replacement